Mathlib/MeasureTheory/Function/UnifTight.lean

```diff
@@ -174,7 +174,7 @@ private theorem unifTight_fin (hp_top : p ≠ ∞) {n : ℕ} {f : Fin n → α 
     obtain ⟨S, hμS, hFε⟩ := h hgLp hε
     obtain ⟨s, _, hμs, hfε⟩ :=
       (hfLp (Fin.last n)).exists_eLpNorm_indicator_compl_lt hp_top (coe_ne_zero.2 hε.ne')
-    refine ⟨s ∪ S, (by measurability), fun i => ?_⟩
+    refine ⟨s ∪ S, measure_union_ne_top hμs.ne hμS, fun i => ?_⟩
```

Comment:
I think the proper proof of this is `by finiteness`: #30967 makes this possible. Hopefully, that PR can get merged quickly, and this one can land afterwards.